### PR TITLE
Drug Expansion

### PR DESCRIPTION
**Pull request by Sypsoti**
Follow-up to snorting mechanics, a whole host of new drugs initially on world server with tweaks (and renaming). Adds the regulations for the drugs to the in-game Reg book, completely reworks hallucination code, and adds nutmeg as the source of DMT (for partially funny reasons), and adjusts recipes. **Make sure to update the wiki as some existing chemical recipes have changed.** Specifically hyperzine. I can do this, but noting for downstream's sake too.

Hallucination code no longer sends pun pun to beat your ass, it now creates hallucinations that talk to you, chat messages, harmless sounds, and with a high enough dose, scary images. This is more beneficial to roleplay than "Help! I'm being beaten to death!"

The drugs have been tested to not murderize you for just taking them, but further balancing will likely be needed.

Also adds a new contraband loadout tab, to be followed up with the rest of discussed contraband from the staff meeting.

Edit: Also adds some creepy sounds for hallucinations courtesy of Serithi.

xoxo

### CODE REVIEW

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -342,3 +341,64 @@
...
+//// Recreational drugs for loadout
+/obj/item/reagent_containers/pill/bliss
+	name = "mysterious pill"
+	desc = "You're not quite sure what's in this..."
+	icon_state = "pill1"
+
+/obj/item/reagent_containers/pill/bliss/Initialize()
+	. = ..()
+	reagents.add_reagent("bliss", 10)
+	color = reagents.get_color()
```
Comment (Atermonera) [inline code/modules/reagents/reagent_containers/pill.dm:354]: You could subtype pill to unidentified, then subtype that to each individual drug, such that you only have to define this once, and then each individual drug defines the initialize() call to add that drug specifically.
E.g.,
```suggestion
/obj/item/reagent_containers/pill/unidentified
	name = "mysterious pill"
	desc = "You're not quite sure what's in this..."
	icon_state = "pill1"


/obj/item/reagent_containers/pill/unidentified/bliss/Initialize()
	. = ..()
	reagents.add_reagent("bliss", 10)
	color = reagents.get_color()
```
(And repeat _just_ the initialize definitions for all the other drugs you're adding)

**Comment by Serithi** [comment]
Woke smeed erryday.

**Comment by KillianKirilenko** [comment]
hallucinations reworked into more than just meme beatdowns???

this truly is the bizarro timeline

(good work though!)

File: code/modules/flufftext/Hallucination.dm [hunk 2]
```diff
@@ -1,223 +1,72 @@
...
+		if(prob(25))
+			switch(chosenhal)
+				if("messages")
+					var/list/msg_list = list("Everything around you feels like it's breathing...",
+					"There are such strange patterns everywhere... you can't take your eyes off them.",
+					"You feel in tune with your surroundings. You're part of them...",
+					"What was that noise?",
+					"You feel like something is crawling on you...!")
+					var/msg = pick(msg_list)
+					to_chat(src, "<span class='warning'>[msg]</span>")
```
Comment (Atermonera) [inline code/modules/flufftext/Hallucination.dm:30]:
```suggestion
					to_chat(src, "<span class='warning'>[pick(msg_list)]</span>")
```
nit: you do this in the other two adjacent spots.

**Review by Atermonera — DISMISSED** [review]

**Review by Atermonera — APPROVED** [review]

**Pull request merged**